lumicks/pylake/channel.py

```diff
@@ -131,6 +135,58 @@ def downsampled_over(self, range_list, reduce=np.mean, where='center'):
 
         return Slice(TimeSeries(d, t), self.labels)
 
+    def downsampled_to(self, frequency, reduce=np.mean, where='center', method="safe"):
+        """Return a copy of this slice downsampled to a specified frequency
+
+        Parameters
+        ----------
+        frequency : int
+            The desired downsampled frequency downsampled (Hz)
+        reduce : callable
+            The `numpy` function which is going to reduce multiple samples into one.
+            The default is `np.mean`, but `np.sum` could also be appropriate for some
+            cases, e.g. photon counts.
+        where : str
+            Where to put the final time point.
+            'center' time point is put at (start + stop) / 2
+            'left' time point is put at start
+        method : str
+            How to handle non-integer frequency ratio.
```

Comment:
Maybe express it in sample time (since that's what we are looking at for the criteria).
```
How to handle target sample times that are not exact multiples of the current sample time.

'safe' new sample time must be an exact multiple of the current sample time, else an exception is raised.
'ceil' rounds the sample rate up to the nearest frequency which fulfills this condition
'force' downsample data with the target input frequency; this will result in variable sample times and a variable number of sampling contributing to each target sample, but must be used for variable-frequency data
```